src/engine.cc

```diff
@@ -132,9 +132,8 @@ void EngineController::UpdateFromUciOptions() {
     if (!syzygy_tb_->init(tb_paths)) {
       CERR << "Failed to load Syzygy tablebases!";
       syzygy_tb_ = nullptr;
-    } else {
-      tb_paths_ = tb_paths;
     }
+    tb_paths_ = tb_paths;
   }
```

Comment:
Should there be an else block here as well where syzygy_tb_ is cleared to nullptr and tb_paths_ is updated back to empty?